out/extension.js

```diff
@@ -220,9 +243,14 @@ function activate(context) {
     }));
 
     // Set up listener for active editor changing
-    context.subscriptions.push(vscode.window.onDidChangeActiveTextEditor( () => {
-        if (vscode.window.activeTextEditor && isSplunkFile(vscode.window.activeTextEditor.document.fileName)) {
+    context.subscriptions.push(vscode.window.onDidChangeActiveTextEditor( async () => {
+        if (!vscode.window.activeTextEditor) {
+            return;
+        }
+        if (isSplunkFile(vscode.window.activeTextEditor.document.fileName)) {
```

Comment:
nit: similar logic to 230, maybe refactor into a method?

Reply:
sorry in the js world, a function 😓

Reply:
yup good call this can be a new `isSpl2File()` function. js has methods too! https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Method_definitions